packages/main/src/Toolbar.ts

```diff
@@ -333,6 +333,8 @@ class Toolbar extends UI5Element {
 	onBeforeRendering() {
 		this.detachListeners();
 		this.attachListeners();
+
+		this.preprocessItems();
```

Comment:
If I comment this line, the tests still pass. Can we secure it with tests?

Reply:
Thank you for your feedback; good catch ;)